.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
```

You might also need to set this enable beta ecosystems variable to true to make the uv stuff work
```suggestion
version: 2
enable-beta-ecosystems: true
updates:
```